src/lib/routes/util.ts

```diff
@@ -58,6 +58,8 @@ export const handleErrors: (
             return res.status(409).json(error).end();
         case 'FeatureHasTagError':
             return res.status(409).json(error).end();
+        case 'RoleInUseError':
+            return res.status(400).json(error).end();
```

Comment:
I see this switch/case is not fully utilizing the semantics, we could do something like this here:

```ts
...
case 'ValidationError':
case 'BadDataError':
case 'OwaspValidationError':
case 'RoleInUseError':
    return res.status(409).json(error).end();
....
```

Reply:
I think you make a fair point, that it would be easy to mis-use and accidentally introduce bugs. Maybe the error-itself could carry the appropriate status code for that error (after all its just an API supposed to give HTTP responses to the user / computer) and that thus make the whole switch/case thing go away?